test/src/test/java/hudson/PluginSEC925Test.java

```diff
@@ -11,13 +11,15 @@
 import java.util.Arrays;
 import java.util.List;
 import java.util.concurrent.Future;
+import org.junit.Ignore;
 
 //TODO merge it within PluginTest after the security release
 public class PluginSEC925Test {
     
     @Rule
     public JenkinsRule r = new JenkinsRule();
     
+    @Ignore("TODO observed to fail in CI with 404 due to external UC issues")
```

Comment:
Should perhaps get a workaround applied similar to https://github.com/jenkinsci/jenkins/pull/3962?

Reply:
Possibly. Does not look like I thought to record a copy of the failure message. Probably best to rewrite tests like these to not attempt to make an Internet connection to begin with, and instead use some hard-coded example JSON data.